### DISCUSSION TITLE
Import JSLT fails

### DISCUSSION THREAD

**Discussion by cvsudheer108** [schibsted/jslt#229] [General]
We are invoking JSLT template from Camel:
        from("direct:logic1-jslt")
                .doTry()
                .log("Step 1")
                .setHeader("CamelJsltResourceUri").constant("mapping/template1.jslt")
                .to("jslt:dummy?allowTemplateFromHeader=true&allowContextMapAll=true")
                .log("StepStep1-jslt o/p: ${in.body}")
Now I have import inside template1.jslt:
import "util.jslt" as util
let obj = util:doSomething()

I am getting error: Cannot load resource 'util.jslt': not found
I tried : import "./util.jslt" as util. , but in vain.

Please let me know if you need more details.

**Comment by larsga** [comment]
By default module files are loaded from the classpath. So if you have `util.jslt` on the classpath this should work.

The API allows you to plug in code that resolves the names of module files into actual files, but whether that is exposed by Camel I don't know.

**Reply by cvsudheer108** [reply]
I have placed the util.jslt in the same folder as the parent jslt. (under resources folder)
I guess Camel doesn't support that as of now... as I don't see it being explicitly mentioned.
https://camel.apache.org/components/3.11.x/jslt-component.html
